src/txn/txn_rollback_to_stable.c

```diff
@@ -276,14 +274,10 @@ __txn_abort_newer_updates(
 			    __wt_page_is_modified(ref->page));
 			local_read = true;
 		}
-		if (ref->page_las->max_timestamp > rollback_timestamp)
-			ref->page_las->max_timestamp = rollback_timestamp;
-		if (ref->page_las->unstable_durable_timestamp >
-		    rollback_timestamp)
-			ref->page_las->unstable_durable_timestamp =
-			    rollback_timestamp;
-		if (ref->page_las->unstable_timestamp > rollback_timestamp)
-			ref->page_las->unstable_timestamp = rollback_timestamp;
+		if (page_las->max_ondisk_ts > rollback_timestamp)
```

Comment:
Please merge this condition with the above one as both are same.
Also if condition at line 279, better write in reverse order ( i.e if (rollback_timestamp < page_las->min_skipped_ts)), for better readability.